sklearn/tests/test_multioutput.py

```diff
@@ -380,6 +381,8 @@ def test_classifier_chain_fit_and_predict_with_logistic_regression():
     assert_equal([c.coef_.size for c in classifier_chain.estimators_],
                  list(range(X.shape[1], X.shape[1] + Y.shape[1])))
 
+    assert isinstance(classifier_chain, ClassifierMixin)
```

Comment:
Not in a test assertion. The point here is that it also ensures a default implementation is score